### ISSUE TITLE
Couldn't decode base64 image, or couldn't write to temp file

### ISSUE THREAD

**Issue by wsippel** [benrugg/AI-Render#19]
### Describe the bug

The error in the title is all I get, using a local Automatic1111 install (updated at the time of writing). The webui terminal output shows that an image does get generated. 

### To reproduce

Just trying to render anything with AI render enabled causes the issue.

### Error log

```
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 127.0.0.1:7860
DEBUG:urllib3.connectionpool:http://127.0.0.1:7860 "POST /sdapi/v1/img2img HTTP/1.1" 200 216413
AI Render Error:  Couldn't decode base64 image, or couldn't write to temp file
```

### Environment

- Blender version (upper right corner of splash screen): 3.3.1
- AI Render version (find in Preferences > Add-ons): 0.4.0
- Operating system (Windows/Mac/Linux): EndeavourOS Linux

### Additional information

The AI Render node appears muted and, if I unmute it manually, automatically mutes itself the moment I start a render, but I assume that's intended behavior?

**Comment by Imyamoe** [comment]
Same issue

**Comment by benrugg** [comment]
I'll check into this right now. @Imyamoe are you also on Linux?

**Comment by Imyamoe** [comment]
> I'll check into this right now. @Imyamoe are you also on Linux?

Sorry, Win10

**Comment by benrugg** [comment]
Ok, gotcha, thanks

**Comment by benrugg** [comment]
Yep, I'm getting that on my Mac as well, so it looks like Automatic1111's response has changed. I should be able to get it fixed quickly.

**Comment by benrugg** [comment]
Ok, fixed in 524501b95da92a8bf6ad11ef1ac0b24ca445c907. Grab the latest release here: https://github.com/benrugg/AI-Render/releases/tag/v0.4.1, or do 'Update' in the add-on preferences.
